Content.Server/Prayer/PrayerSystem.cs

```diff
@@ -69,7 +69,7 @@ public void SendSubtleMessage(IPlayerSession target, string messageString, strin
             return;
 
         _popupSystem.PopupEntity(popupMessage, target.AttachedEntity.Value, Filter.Empty().AddPlayer(target), PopupType.Large);
-        _chatManager.ChatMessageToOne(ChatChannel.Local, messageString, popupMessage + " \"{0}\"", EntityUid.Invalid, false, target.ConnectedClient);
+        _chatManager.ChatMessageToOne(ChatChannel.Local, messageString, popupMessage + $" \"{messageString}\"", EntityUid.Invalid, false, target.ConnectedClient);
```

Comment:
Why is messagestring being passed twice.

Reply:
i think i'm stupid